detection_rules/eswrap.py

```diff
@@ -233,12 +233,17 @@ def normalize_file(events_file):
 @click.option('--cloud-id', callback=set_param_values, expose_value=True)
 @click.option('--user', '-u', callback=set_param_values, expose_value=True, hide_input=False)
 @click.option('--password', '-p', callback=set_param_values, expose_value=True, hide_input=True)
-def kibana_upload(toml_files, kibana_url, cloud_id, user, password):
+@click.option('--space', callback=set_param_values, expose_value=True, hide_input=False)
+def kibana_upload(toml_files, kibana_url, cloud_id, user, password, space):
     """Upload a list of rule .toml files to Kibana."""
     from uuid import uuid4
     from .packaging import manage_versions
     from .schemas import downgrade
 
+    # Update kibana url if a space is defined
+    if space:
+        kibana_url = "{}/s/{}".format(kibana_url, space)
```

Comment:
Thanks @SHolzhauer! Can you actually move this inside the `Kibana` class? I think that's the better place to put it, since that could be used in other contexts, too.

Reply:
Indeed seems more appropriate to put it, I have moved it @rw-access